packages/craco/lib/cra.js

```diff
@@ -22,15 +22,15 @@ const craPaths = require(resolveConfigFilePath("paths.js"));
 /************  Webpack Dev Config  *******************/
 
 function getWebpackDevConfigPath() {
-    return resolveConfigFilePath("webpack.config.dev");
+    return resolveConfigFilePath("webpack.config");
```

Comment:
Do you think it's better to check react-scripts version first?
If it's <= v2.1.1, import as is, if not, import using the new approach!

Reply:
definitely check the version of react-scripts or `try/catch` with [`require.resolve`](https://nodejs.org/api/modules.html#modules_require_resolve_request_options)

Reply:
May this PR for `react-app-rewired` helps:
https://github.com/timarney/react-app-rewired/pull/344/files

Reply:
Nice feedback, @AndyOGo 
Maybe I think we should check the version because each version has a specific change on webpack.config.

Reply:
I wonder if craco could just have a version compatibility table, instead of trying to support every version of `react-scripts`.

E.g. The next version of `craco` could only support `react-scripts >= 2.1.2`, and you have to use an earlier version of `craco` for older versions of `react-scripts`. 

Usually it's good to provide backwards compatibility, but I think this case is different. Everyone should be trying to use the latest version of `react-scripts`, and if you don't update `react-scripts`, then there's no reason to update `craco`. Also you don't need to change your webpack config very often.

So I think it's fine to have a breaking change here, and just follow the `react-scripts` updates.

Reply:
fair point

Reply:
I agree with @ndbroadbent.

I tryed to make it backward compatible for a few minor changes but it get quite hard to maintain.

I'll be glad to accept a PR if someone want to submit a first draft of a version compatibility table.